### PR TITLE
Dribbble Presence

### PR DESCRIPTION
**Pull request by Vimeux**
Hi there,

I have created a presence for the [Dribbble](https://dribbble.com/) site.
This is my first commit here, I tested it on Chrome, Safari and Firefox, and it works perfectly.
If there are things to review, let me know, I'll take care of it.

See you soon.

### CODE REVIEW

**Comment by EncryptedDev** [comment]
Please format your code by running `yarn lint` (you will need both `eslint` and `prettier` installed globally for this to work)

File: websites/D/Dribbble/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
```
Comment (EncryptedDev) [inline websites/D/Dribbble/dist/metadata.json:2]: Outdated schema version
```suggestion
  "$schema": "https://schemas.premid.app/metadata/1.5",
```

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/D/Dribbble/presence.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    presenceData.state = `- ${profileName} -`;
+  } else presenceData.details = "Homepage";
+
+  if (document.body.id === "user-profile") {
+    // view personal profile
+    const profilePath = path,
+      pathArray = profilePath.split("/"),
+      [, profileName] = pathArray;
+    presenceData.details = "Viewing personal profile :";
+    presenceData.state = `- ${profileName} -`;
```
[the remark this answers is not in the document]
Comment (Vimeux) [reply websites/D/Dribbble/presence.ts:75]: These lines allow me to get the name of the user in the URL to display it in the presence.

[the remark this answers is not in the document]
Comment (EncryptedDev) [reply websites/D/Dribbble/presence.ts:75]: I understand that, but you are just adding it to a string so the outcome would be `- Encrypted -`

**Comment by EncryptedDev** [comment]
Upload proof of the presence working

**Comment by Vimeux** [comment]
Capture d’écran 2021-11-18 à 18 15 27

Capture d’écran 2021-11-18 à 18 16 40

Capture d’écran 2021-11-18 à 18 17 40

Capture d’écran 2021-11-18 à 18 18 04

**Review by EncryptedDev — APPROVED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Pull request merged**